tempodb/encoding/vparquet2/block_search.go

```diff
@@ -229,7 +225,7 @@ func makePipelineWithRowGroups(ctx context.Context, req *tempopb.SearchRequest,
 			max = (time.Millisecond * time.Duration(req.MaxDurationMs)).Nanoseconds()
 		}
 		durFilter := pq.NewIntBetweenPredicate(min, max)
-		traceIters = append(traceIters, makeIter("DurationNano", durFilter, "Duration"))
+		traceIters = append(traceIters, makeIter("DurationNanos", durFilter, "Duration"))
```

Comment:
What was the rationale for this change?  It looks like the column is really named [DurationNano](https://github.com/grafana/tempo/blob/main/tempodb/encoding/vparquet2/schema.go#L183) and this change will not work.

Reply:
oopps, copy paste error. reverting back.

Reply:
> The wal metrics lgtm, but I would like @stoewer to review the vparquet2 DurationNanos change.

cc @stoewer